src/cider/piggieback_impl.clj

```diff
@@ -203,12 +211,45 @@
                    (readers/source-logging-push-back-reader
                     (java.io.StringReader. form-str))))))
 
+(defn- wrap-pprint [form]
```

Comment:
We should add some explanation about the naming here, as `wrap-something` is the standard convention for naming middleware functions in nREPL and this might confuse some people.

Reply:
> We should add some explanation about the naming here, as `wrap-something` is the standard convention for naming middleware functions in nREPL and this might confuse some people.

Added a docstring to clarify that this wraps an s-expression with `cljs.pprint/pprint`. We can also change the name to something else if you prefer.